packages/commentor/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+pkgname=commentor
+pkgver=19.d81a660
+pkgrel=1
+pkgdesc='Extract all comments from the specified URL resource.'
+groups=('blackarch' 'blackarch-code-audit')
+arch=('any')
+url='https://github.com/D3vil0p3r/comMENTOR'
+license=('GPL3')
+depends=('curl' 'tidy' 'xmlstarlet')
+makedepends=('git')
+source=("git+https://github.com/D3vil0p3r/$pkgname.git")
+sha512sums=('SKIP')
+pkgver() {
+  cd $pkgname
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  mkdir -p "${pkgdir}/usr/share/$pkgname"
```

Comment:
only use `$pkgdir` and not `${pkgdir}` when it's not needed, also use install and not mkdir eg. `install -dm 755 "$pkgdir/usr/bin"`